Source/Scene/GoogleEarthEnterpriseMapsProvider.js

```diff
@@ -91,14 +94,14 @@ define([
      *
      *
      * @example
-     * var google = new Cesium.GoogleEarthImageryProvider({
+     * var google = new Cesium.GoogleEarthEnterpriseMapsProvider({
      *     url : 'https://earth.localdomain',
      *     channel : 1008
      * });
      *
      * @see {@link http://www.w3.org/TR/cors/|Cross-Origin Resource Sharing}
      */
-    function GoogleEarthImageryProvider(options) {
+    function GoogleEarthEnterpriseMapsProvider(options) {
```

Comment:
For the old file, you just need to keep the constructor function and have it return an instance of the new version.  No need to have a completely duplicate of the code.